src/upload/assets.rs

```diff
@@ -183,11 +266,22 @@ pub fn get_updated_metadata(metadata_file: &str, media_link: &str) -> Result<Str
 
     for file in &mut metadata.properties.files {
         if file.uri.eq(&metadata.image) {
-            file.uri = media_link.to_string();
+            file.uri = image_link.to_string();
         }
     }
 
-    metadata.image = media_link.to_string();
+    metadata.image = image_link.to_string();
+
+    if let Some(animation) = animation_link.clone() {
```

Comment:
The `properties.files` is not enforced and it may go away in the future.  Perhaps we can do the same approach as the image (for loop above).